### PR TITLE
Con 3509 agreement not signed

### PR DESCRIPTION
**Pull request by VasanthaKasirajan3008**

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/LegalEntitySignedAgreementViewModel.cs [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
-
-namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
+﻿namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
 {
     public class LegalEntitySignedAgreementViewModel
     {
-        public string HashedAccountId { get; set; }
+        public string AccountHashedId { get; set; }
         public string TransferConnectionCode { get; set; }
         public string LegalEntityCode { get; set; }
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/LegalEntitySignedAgreementViewModel.cs:7]: what is legal entity code?

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/LegalEntitySignedAgreementViewModel.cs [hunk 2]
```diff
@@ -1,19 +1,13 @@
...
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
+﻿namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
 {
     public class LegalEntitySignedAgreementViewModel
     {
-        public string HashedAccountId { get; set; }
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/LegalEntitySignedAgreementViewModel.cs:10]: I actually prefer `HashedAccountId` is this change just for consistency?

**Review by cofaulco — COMMENTED** [review]
minor comments

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/LegalEntitySignedAgreementViewModel.cs (hunk 1 above)
Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/LegalEntitySignedAgreementViewModel.cs:7]: Legal Entity Code  comes from  [employer_account].LegalEntity table

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/LegalEntitySignedAgreementViewModel.cs (hunk 2 above)
Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/LegalEntitySignedAgreementViewModel.cs:10]: yes changed  for consistency :-)

**Review by cofaulco — APPROVED** [review]

**Pull request merged**